mkdocs.yml

```diff
@@ -128,6 +129,7 @@ nav:
           - Practical Batch Array Jobs: cheaha/slurm/practical_sbatch.md
           - Managing Jobs: cheaha/slurm/job_management.md
           - GPUs: cheaha/slurm/gpu.md
+          - Slurm Tutorial: cheaha/slurm/slurm_tutorial.md
```

Comment:
Move this up to be between "SLURM/Introduction" and "SLURM/Submitting Jobs". Let's give it a little more visibility :)

Reply:
Sure. Done.